src/scripts/main.js

```diff
@@ -354,7 +354,40 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
+const peopleTableData = [...people];
 
-// write your code here
+const fragment = document.createDocumentFragment();
+
+peopleTableData.forEach((person) => {
+  // Calculate age and century
+  person.age = person.died - person.born;
+  person.century = Math.ceil(person.died / 100);
+
+  // Convert sex to readable format
+  switch (person.sex) {
+    case 'm':
+      person.sex = 'Male';
+      break;
+    case 'f':
+      person.sex = 'Female';
+  }
```

Comment:
The switch statement for converting 'sex' to a readable format does not handle unexpected values. Consider adding a default case to handle any unexpected 'sex' values that might not be 'm' or 'f'.